src/engine/source/json/src/json.cpp

```diff
@@ -595,15 +595,11 @@ bool Json::isInt64(std::string_view path) const
         {
             return value->IsInt64();
         }
-        else
```

Comment:
The unit tests were not affected, this is probably because we do not have unit tests for this function. Add the 4 cases:
- The path does not exist
- The path is invalid 
- Tje path is valid and exist, but not a int
- Tje path is valid and exist, and is a int.